packages/contracts-bedrock/scripts/go-ffi/utils.go

```diff
@@ -127,3 +128,38 @@ func (n *proofList) Put(key []byte, value []byte) error {
 func (n *proofList) Delete(key []byte) error {
 	panic("not supported")
 }
+
+// encodeGasPayingToken encodes a gas paying token into a byte array.
+func encodeGasPayingToken(token common.Address, decimals uint8, name common.Hash, symbol common.Hash) ([]byte, error) {
+	arguments := abi.Arguments{
```

Comment:
to follow the pattern in this file, i recommend declaring the arguments as a global variable, then we can delete these helper functions